draft-ietf-quic-transport.md

```diff
@@ -1184,7 +1203,9 @@ endpoint cannot send a frame of a type that is unknown to its peer.
 An extension to QUIC that wishes to use a new type of frame MUST first ensure
 that a peer is able to understand the frame.  An endpoint can use a transport
 parameter to signal its willingness to receive one or more extension frame types
-with the one transport parameter.
+with the one transport parameter. Otherwise it is assumed that enpoints have
```

Comment:
I'm not sure we want to imply or endorse the use of out of band negotiation, so I'd stick with the existing text.